actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -47,7 +47,7 @@ func (a Actor) Constructor(rt vmr.Runtime, rootKey *addr.Address) *adt.EmptyValu
 }
 
 type AddVerifierParams struct {
-	Address   addr.Address
+	Address   addr.Address // PARCHECK: could be a bogus address
```

Comment:
This will be solved along with https://github.com/filecoin-project/specs-actors/issues/556.